pkg/migration/migrations/separated_intents.go

```diff
@@ -562,6 +562,18 @@ func postSeparatedIntentsMigration(
 				return err
 			})
 			if err != nil {
+				// TODO(erikgrinaker): This should be moved into a common helper for
+				// all migrations that e.g. manages checkpointing and retries as well.
+				// We add a message here for 21.2 temporarily, since this has been seen
+				// to fail in the wild.
+				log.Warningf(ctx, `Migrate command failed for range %d: %s.
+Command must be applied on all range replicas (not just a quorum). Please make
+sure all ranges are healthy and fully upreplicated. Heavy rebalancing may
+interfere with the migration, consider temporarily disabling rebalancing with
+these cluster settings:
+kv.allocator.load_based_rebalancing=1
+kv.allocator.range_rebalance_threshold=1`,
+					desc.RangeID, err)
```

Comment:
Perhaps return errors.WithHint to include above suggestion?
Also, should you mention the timeout setting you've added?

Reply:
>Perhaps return errors.WithHint to include above suggestion?

You mean in addition to the `log.Warningf()`? Will that make it e.g. show up in job summaries and such?

>Also, should you mention the timeout setting you've added?

Might as well. I doubt it'll help if it doesn't succeed within a minute, but who knows.